kumascript/macros/spec2.ejs

```diff
@@ -27,97 +27,109 @@ status['ES8'] = status['ES2017'];
   'REC': mdn.localString({
     'en-US' : 'Recommendation',
-    'ja'    : '勧告',
-    'ru'    : 'Рекомендация',
     'fr'    : 'Recommendation',
-    'pt-BR' : 'Recomendação'
+    'ja'    : '勧告',
+    'ko'    : '권고안',
+    'pt-BR' : 'Recomendação',
+    'ru'    : 'Рекомендация'
   }),
   'PR' : mdn.localString({
     'en-US' : 'Proposed Recommendation',
-    'ja'    : '勧告案',
-    'ru'    : 'Предложенная рекомендация',
     'fr'    : 'Recommendation proposée',
-    'pt-BR' : 'Recomendação proposta'
+    'ja'    : '勧告案',
+    'ko'    : '제안 권고안',
+    'pt-BR' : 'Recomendação proposta',
+    'ru'    : 'Предложенная рекомендация'
   }),
   'CR' : mdn.localString({
     'en-US' : 'Candidate Recommendation',
-    'ja'    : '勧告候補',
-    'ru'    : 'Кандидат в рекомендации',
     'fr'    : 'Candidat au statut de recommandation',
-    'pt-BR' : 'Candidata a Recomendação'
+    'ja'    : '勧告候補',
+    'ko'    : '후보 권고안',
+    'pt-BR' : 'Candidata a Recomendação',
+    'ru'    : 'Кандидат в рекомендации'
   }),
   'RC' : mdn.localString({
     'en-US' : 'Release Candidate',
-    'ja'    : 'リリース候補',
-    'ru'    : 'Кандидат в рекомендации',
     'fr'    : 'En cours de validation comme recommandation',
-    'pt-BR' : 'Recomendação em homologação'
+    'ja'    : 'リリース候補',
+    'ko'    : '최종 후보',
+    'pt-BR' : 'Recomendação em homologação',
+    'ru'    : 'Кандидат в рекомендации'
   }),
   'WD' : mdn.localString({
     'en-US' : 'Working Draft',
-    'ja'    : '草案',
-    'ru'    : 'Рабочий черновик',
     'fr'    : 'Version de travail',
-    'pt-BR' : 'Rascunho atual'
+    'ja'    : '草案',
+    'ko'    : '초안',
+    'pt-BR' : 'Rascunho atual',
+    'ru'    : 'Рабочий черновик'
   }),
   'ED' : mdn.localString({
     'en-US' : 'Editor\'s Draft',
-    'ja'    : '編集者草案',
-    'ru'    : 'Редакторский черновик',
     'fr'    : 'Brouillon de l\'éditeur',
-    'pt-BR' : 'Rascunho editorial'
+    'ja'    : '編集者草案',
+    'ko'    : '작성 중 초안',
+    'pt-BR' : 'Rascunho editorial',
+    'ru'    : 'Редакторский черновик'
   }),
   'Old-Transforms' : mdn.localString({
     'en-US' : 'This has been merged in another draft. Please update.',
-    'ja'    : 'この仕様は他のドラフトへ移動されました。アップデートしてください。',
-    'ru'    : 'Эта спецификация была слита с другим черновиком. Пожалуйста, обновите.',
     'fr'    : 'Cette spécification a été fusionnée avec une autre. Veuillez mettre à jour cette mention.',
-    'pt-BR' : 'Isso foi anexado a outro rascunho. Por favor, atualize.'
+    'ja'    : 'この仕様は他のドラフトへ移動されました。アップデートしてください。',
+    'ko'    : "다른 초안과 합쳐졌습니다. 업데이트 하세요.",
+    'pt-BR' : 'Isso foi anexado a outro rascunho. Por favor, atualize.',
+    'ru'    : 'Эта спецификация была слита с другим черновиком. Пожалуйста, обновите.'
   }),
   'Living' : mdn.localString({
     'en-US' : 'Living Standard',
-    'ja'    : '現行の標準',
-    'ru'    : 'Живой стандарт',
     'fr'    : 'Standard évolutif',
-    'pt-BR' : 'Padrão em tempo real'
+    'ja'    : '現行の標準',
+    'pt-BR' : 'Padrão em tempo real',
```

Comment:
There is no Korean translation here. Is this intentional?

Reply:
yes, there is no Korean word that has same meaning as living standard. We can directly translate it, but it would make it more confusing, so decided to leave it as it is.

Reply:
Makes sense, thanks!